acceptancetests/assess_remove_machine.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import logging
+import typing
+import yaml
+
+from deploy_stack import BootstrapManager
+from utility import add_basic_testing_arguments, add_arg_juju_bin, configure_logging
+
+Optional = typing.Optional
+log = logging.getLogger("remove_machine")
+
+if __name__ != "__main__":
```

Comment:
Why change from the common main entry point?

Reply:
Sorry, should have used the project's conventions. I like to abort early because it avoids wasted work when the file is imported.